lib/pbench/agent/tool_meister.py

```diff
@@ -186,6 +236,18 @@ def start(self):
 
     def stop(self):
         if not self.failure:
+            if self.name == "pcp":
```

Comment:
Could this code be moved into the `terminate` and `wait` methods?  Is there a reason it is separate?

Reply:
Not sure which terminate and wait methods you are referring to, the PersistentTool class only has a start and stop method

Reply:
Seems like they need to be added so that you can abstract away the details of how each type of tool is stopped.

Reply:
Will be fixed in separate podman modularization PR if it still makes sense after PR #2005